pygmt/helpers/utils.py

```diff
@@ -188,30 +188,36 @@ def _check_encoding(
 
 
 def data_kind(
-    data: Any = None, required: bool = True
```

Comment:
The default value of `data` is `None`, but it makes no sense to call `data_kind()` without any arguments. Here I make `data` a required argument.